tests/mbedtls_test/Makefile

```diff
@@ -0,0 +1,42 @@
+APPLICATION= lwip_mbedtls_test
+
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD ?= native
+
+USEPKG += mbedtls
+USEMODULE += mbedtls_entropy
+USEMODULE += ipv4_addr
+USEMODULE += lwip_arp
+USEMODULE += lwip_ipv4
+USEMODULE += lwip_ipv6
+USEMODULE += lwip_dhcp_auto
+USEMODULE += lwip lwip_netdev
+USEMODULE += lwip_udp
+USEMODULE += lwip_tcp
+USEMODULE += sock_tcp
+
+# Add also the shell, some shell commands
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+# Add atwinc15x0 driver
+USEMODULE += atwinc15x0
```

Comment:
No need to force the use of this driver, `sam0_eth`, `netdev_tap` or `esp_wifi` would work as well - just use `netdev_default`